### PR TITLE
DL-3891 play 27 upgrade for vat-frontend

### PR DESCRIPTION
**Pull request by emmaatkins**

### CODE REVIEW

File: app/connectors/Vrn.scala [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/*
```
Comment (benryan1790) [inline app/connectors/Vrn.scala:1]: I think this file is in the wrong package. Shoudl be in models or connectors/models

File: app/services/VatPartialBuilder.scala [hunk 2]
```diff
@@ -41,21 +41,25 @@ class VatPartialBuilderImpl @Inject()(val enrolmentsStore: EnrolmentsStoreServic
...
+
   override def buildReturnsPartial(
                                     vatData: VatData, vatEnrolment: VatEnrolment
                                   )(
                                     implicit request: AuthenticatedRequest[_], messages: Messages
                                   ): Html =
     vatData.returnsToCompleteCount match {
       case Some(0) =>
-        Logger.warn("Showing no returns card")
+        val logger: Logger = Logger(this.getClass)
```
Comment (benryan1790) [inline app/services/VatPartialBuilder.scala:53]: could we use one logger val here?

File: app/services/VatService.scala [hunk 3]
```diff
@@ -72,12 +73,14 @@ class VatService @Inject()(vatConnector: VatConnector)(implicit ec: ExecutionCon
         val directDebitStatus = DirectDebitStatus.from(calendarData.directDebit)
         Some(CalendarDerivedInformation(Calendar(frequency, directDebitStatus),calendarData.countReturnsToComplete))
       case _ =>{
-        Logger.warn("Received None for calendar")
+        val logger: Logger = Logger(this.getClass)
```
Comment (benryan1790) [inline app/services/VatService.scala:76]: could we use one logger val here?

File: app/connectors/Vrn.scala (hunk 1 above)
Comment (CEPoole) [reply app/connectors/Vrn.scala:1]: models, not connectors/models pls

**Comment by CEPoole** [comment]
Looks like duplicate commit messages - is it worth squashing these into one commit?

**Pull request merged**